joss/paper.md

```diff
@@ -38,7 +38,7 @@ bibliography: paper.bib
 
 # Summary
 
-Any mass concentration in the Universe, luminous or dark, from vast galaxy clusters to stars within galaxies, can be studied through its gravitationally induced deflections of light rays from background sources – a phenomenon known as Strong Gravitational Lensing (SGL). This has several cutting-edge applications, for example: measuring the Hubble constant and shedding more light into the apparent tension between early and late Universe, detecting the presence of massive subhalos within distant galaxies that can constrain different dark matter models, and studying a galaxy’s mass partition between baryons and dark matter with direct implications on galaxy evolution.
+Any mass concentration in the Universe, luminous or dark, from vast galaxy clusters to stars within galaxies, can be studied through its gravitational deflection of light rays from background sources. This phenomenon, in its most impressive regime, is known as Strong Gravitational Lensing (SGL). It has several cutting-edge applications, for example: measuring the Hubble constant and shedding more light into the apparent tension between early and late Universe, detecting the presence of massive subhalos within distant galaxies that can constrain different dark matter models, and studying a galaxy’s mass partition between baryons and dark matter with direct implications on galaxy evolution.
```

Comment:
@gvernard What about this? Just to make sure we don't entirely ignore other GL regimes...